solr/core/src/java/org/apache/solr/search/facet/FacetRangeProcessor.java

```diff
@@ -112,6 +137,7 @@ public void process() throws IOException {
     } else {
       // phase#1: build list of all buckets and return full facets...
       createRangeList();
+      sortRangeList();
```

Comment:
Actually let me put this being the flag so that no changes occur when flag is off.